base/pom.xml

```diff
@@ -20,6 +20,11 @@
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>${version.commons.codec}</version>
```

Comment:
Move it to the parent pom

Reply:
I just removed the version - that should do. 41d33a11